fineract-provider/src/main/java/org/apache/fineract/notification/domain/Notification.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.notification.domain;
+
+import org.apache.fineract.infrastructure.core.domain.AbstractPersistableCustom;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "notification_generator")
+public class Notification extends AbstractPersistableCustom<Long> {
+
+    @Column(name = "object_type")
+    private String objectType;
+
+    @Column(name = "object_identifier")
+    private Long objectIdentifier;
+
+    @Column(name = "action")
+    private String action;
+
+    @Column(name = "actor")
+    private Long actorId;
```

Comment:
Sorry about that, when the last time I changed the actorId to Long, I forgot to change it in the notification_generator definition.